packages/sync/package.json

```diff
@@ -55,6 +56,7 @@
     "apollo-link-retry": "2.2.13",
     "apollo-link-ws": "1.0.17",
     "apollo-upload-client": "10.0.0",
+    "crypto-js": "^3.1.9-1",
```

Comment:
@StephenCoady is this still required?

Reply:
@wei-lee no, I removed it in another commit I forgot to push sorry.